main.go

```diff
@@ -358,15 +368,10 @@ func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
 	}
 
 	if feature.Gates.Enabled(feature.RuntimeSDK) {
-		// This is the creation of the single RuntimeExtension registry for the controller.
-		registry := runtimeregistry.New()
 		if err = (&runtimecontrollers.ExtensionConfigReconciler{
```

Comment:
I would like to preserve the current order in which we setup the reconcilers.

It will be eventually used in multiple reconcilers and our current pattern is to setup shared dependencies above and then use them across the controllers we need them